package ch04.ex05.case01;

public class ExceptionTest {
	public static void main(String[] args) {
		
		ExceptionMaker maker = new ExceptionMaker();
		
		String password = null;		
		try {
			maker.setPassword(password); 
			System.out.println("오류 없음1");
		} catch (PasswordException e) {
			System.out.println(e.getMessage());
		}
		
		password = "";
	}
}
